### PR TITLE
Add rchavarria as maintainer

### PR DESCRIPTION
**Pull request by rchavarria**
I'm not very proud of adding myself as a maintainer, but somebody needs to do the job 😉 

Come on @exercism/ecmascripts, let's prepare for Nextercism.

### CODE REVIEW

**Comment by matthewmorgan** [comment]
@rchavarria are you alumnus-ing yourself into inactive status?  Or do I misunderstand the purpose of the key?

File: config/maintainers.json [hunk 1]
```diff
@@ -12,14 +12,14 @@
       "show_on_website": false
     },
     {
-      "alumnus": false,
+      "alumnus": true,
```
Comment (joelwallis) [inline config/maintainers.json]: [It should be `false`](https://github.com/exercism/ecmascript/pull/357#pullrequestreview-69654498).

**Comment by rchavarria** [comment]
I completely misunderstood the meaning of the `alumnus` key :disappointed: 

I'll fix it.

File: config/maintainers.json (hunk 1 above)
Comment (rchavarria) [reply config/maintainers.json]: Should be fixed now

**Pull request merged**